README.md

```diff
@@ -43,6 +43,8 @@ and merging**:
 
 ## Features and design principles
 
+Automerge is spawn from the [academic essay "Local-first software"](https://www.inkandswitch.com/local-first.html) that compares various approaches to automatic conflict-resolution and details the reasoning behind how automerge came to be.
```

Comment:
The [first commit](3a68032e284b107302a08aa7cf172b8c5816f3f4) in Automerge happened 2 years before that article was published...

Reply:
Huh? It states "Ink & Switch has developed an open-source, JavaScript CRDT implementation called Automerge."

Anyways, one can rephrase the text, but the essay is still really helpful.

```suggestion
Automerge is featured in the [academic essay "Local-first software"](https://www.inkandswitch.com/local-first.html) that compares various approaches to automatic conflict-resolution and explains the workings behind why automerge is a preferable solution.
```